components/eventing-controller/reconciler/backend/reconciler.go

```diff
@@ -548,6 +550,11 @@ func (r *Reconciler) CreateOrUpdatePublisherProxy(ctx context.Context, backend e
 		return nil, fmt.Errorf("unknown eventing backend type %q", backend)
 	}
 
+	if err := r.setPublisherOwnerReference(ctx, desiredPublisher); err != nil {
+		r.Log.Error(err, "setting owner references")
```

Comment:
Since Marco's PR is merged, and it includes updates to how tests log as well, I think all logs need to get updated.

Reply:
Done